kasa/cli.py

```diff
@@ -1051,9 +1064,9 @@ async def turn_on_behavior(dev: SmartBulb, type, last, preset):
 async def update_credentials(dev, username, password):
     """Update device credentials for authenticated devices."""
     # Importing here as this is not really a public interface for now
-    from kasa.tapo import TapoDevice
+    from kasa.smart import SmartDevice
```

Comment:
This can be removed now as we are importing the smartdevice already in the beginning.

Reply:
Done